site/src/app.css

```
@@ -85,7 +85,7 @@
   --sms-max-width: 23em;
   --sms-focus-border: 1px solid var(--link-color);
   --sms-active-color: cornflowerblue;
-  --sms-selected-bg: transparent;
+  --sms-selected-bg: color-mix(in srgb, currentColor 12%, transparent);
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)